micr_XPCTzarr/sub-LADAF-2020-31/sub-LADAF-2020-31_sessions.json

```diff
@@ -0,0 +1,20 @@
+{
+    "session_id": {
+        "Description": "Session ID"
+    },
+    "acq_time": {
+        "Description": "Time when the sample has been acquired"
+    },
+    "proposal_facility": {
+        "Description": "Synchrotron radiation facility where the proposal was submitted"
+    },
+    "proposal_reference": {
+        "Description": "Synchrotron beam time proposal reference"
+    },
+    "proposal_title": {
+        "Description": "Synchrotron beam time proposal title"
+    },
+    "proposal_members": {
+        "Description": "Synchrotron beam time proposal members"
+    }
```

Comment:
@dstansby @Guillaume-Gaisne @ucbpclw
Is it relevant here or at a different level?

Reply:
I think this makes sense as session metadata

Reply:
Note that I have added the session DOI from ESRF